src/components/Pagination/Pagination.jsx

```diff
@@ -18,11 +17,12 @@ const RIGHT_PAGE = 'RIGHT';
 
 const Pagination = props => {
   const theme = useTheme();
+  const mq = window && window.matchMedia('(min-width: 480px)');
```

Comment:
We will need to do a quick check here to make sure window is not undefined before we set the variable:

```js
let mq;

if (typeof window !== 'undefined') {
  mq = window.matchMedia('(min-width: 480px)')
}
```